lapack/gonum/dpstf2.go

```diff
@@ -17,10 +17,19 @@ import (
 // The factorization has the form
 //  Pᵀ * A * P = Uᵀ * U ,  if uplo = blas.Upper,
 //  Pᵀ * A * P = L  * Lᵀ,  if uplo = blas.Lower,
-// where U is an upper triangular matrix and L is lower triangular, and P is
-// stored as vector piv.
+// where U is an upper triangular matrix, L is lower triangular, and P is a
+// permutation matrix.
 //
-// Dpstf2 does not attempt to check that A is positive semidefinite.
+// tol is a user-defined tolerance. The algorithm terminates if the pivot is
+// less than or equal to tol. If tol is negative, then n*eps*max(A[k,k]) will be
```

Comment:
Maybe we don't need to document what value will be used and just say "a default value".

Reply:
I think that's reasonable.

Reply:
I changed it before seeing that you edited the comment. Now I've reverted it back, can you please re-approve?